Locally with,
```
./build.py -f components/blitz/build.xml test
```
I then get,
```
===============================================
blitz.unit
Total tests run: 47, Failures: 1, Skips: 0
===============================================
```
and `components/blitz/target/reports/xml/ome.formats.importer.OMEROWrapperTest_results.xml` reports,
```xml
      <failure type="org.testng.internal.thread.ThreadTimeoutException"
                  message="Method org.testng.internal.TestNGMethod.testMismatchedWrappers() didn&apos;t finish within the time-out 1000"
              >
<![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.testMismatchedWrappers() didn't finish within the time-out 1000
  java.lang.Object.hashCode(Native Method)
  com.esotericsoftware.kryo.util.ObjectMap.push(ObjectMap.java:218)
  com.esotericsoftware.kryo.util.ObjectMap.putResize(ObjectMap.java:182)
  com.esotericsoftware.kryo.util.ObjectMap.resize(ObjectMap.java:488)
  com.esotericsoftware.kryo.util.ObjectMap.put_internal(ObjectMap.java:137)
  com.esotericsoftware.kryo.util.ObjectMap.put(ObjectMap.java:78)
  com.esotericsoftware.kryo.util.DefaultClassResolver.register(DefaultClassResolver.java:50)
  com.esotericsoftware.kryo.util.DefaultClassResolver.registerImplicit(DefaultClassResolver.java:56)
  com.esotericsoftware.kryo.Kryo.getRegistration(Kryo.java:461)
  com.esotericsoftware.kryo.Kryo.getSerializer(Kryo.java:476)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:109)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:378)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:289)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:378)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:289)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:657)
  loci.formats.Memoizer$KryoDeser.loadReader(Memoizer.java:163)
  loci.formats.Memoizer.loadMemo(Memoizer.java:866)
  loci.formats.Memoizer.setId(Memoizer.java:653)
  ome.formats.importer.OMEROWrapperTest.testMismatchedWrappers(OMEROWrapperTest.java:131)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
  org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
  java.util.concurrent.FutureTask.run(FutureTask.java:262)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>
```
With,
```
./components/tools/travis-build java-test
```
I thus get,
```
BUILD FAILED
```

--- a/components/blitz/test/ome/formats/importer/OMEROWrapperTest.java
+++ b/components/blitz/test/ome/formats/importer/OMEROWrapperTest.java
@@ -119,19 +119,17 @@ public class OMEROWrapperTest {
         }
     }
 
-    @Test
+    @Test(invocationCount = 10)
     public void testMismatchedWrappers() throws Exception {
-        for (int i = 0; i < 10; i++) {
-            try (IFormatReader r = new MinMaxCalculator();
-                 Memoizer m = new Memoizer(0L);
-                 Memoizer m1 = new Memoizer(r, 0L)) {
-                m.setId(fake.getAbsolutePath());
-                m.close();
-                Assert.assertTrue(m.isSavedToMemo());
-                m1.setId(fake.getAbsolutePath());
-                Assert.assertFalse(m1.isLoadedFromMemo());
-                m1.close();
-            }
+        try (IFormatReader r = new MinMaxCalculator();
+                Memoizer m = new Memoizer(0L);
+                Memoizer m1 = new Memoizer(r, 0L)) {
+            m.setId(fake.getAbsolutePath());
+            m.close();
+            Assert.assertTrue(m.isSavedToMemo());
+            m1.setId(fake.getAbsolutePath());
+            Assert.assertFalse(m1.isLoadedFromMemo());
+            m1.close();
         }
     }
 

